src/reference/informative/informC02_units_reference.rst

```diff
@@ -8,4 +8,155 @@
 
   .. container:: infospec
 
-    **TODO**
\ No newline at end of file
+    .. container:: heading3
+
+      Understanding units references
+
+    Units are different from other element types in that their names could refer either to other units which you've imported or created in your model, or to one of the list of :ref:`built-in units<table_built_in_units>`.  
+    The same naming conventions apply here as elsewhere, so that part is fairly straightforward.
```

Comment:
I'm guessing you're talking about being a valid identifier here? Not sure this sentence is needed